src/seedu/addressbook/data/person/Address.java

```diff
@@ -9,8 +13,14 @@
 public class Address {
 
     public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses MUST be in 'a/123, Clementi Ave 3, #12-34, 231534'.";
```

Comment:
hmmmm, how are you enforcing this format? how are you checking if the address input is valid?